tests/plan/export/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "cp data/plan.txt ${tmp}/plan.fmf"
+        rlRun "pushd $tmp"
+        rlRun "tmt init"
+        rlRun "output=\$(mktemp)" 0 "Create output file"
+        rlRun "set -o pipefail"
+    rlPhaseEnd
+
+    rlPhaseStartTest
+        rlRun "tmt plan export . | tee $output" 0 "Export plan"
```

Comment:
Using `rlRun -s` together with `rlAssertGrep $rlRun_LOG` is the preffered way to check the output, `beakerlib` takes care of everything.